src/PhappManifest.php

```diff
@@ -135,6 +135,15 @@ public function getName() {
     return $this->config['name'];
   }
 
+  /**
+   * Gets the environment variables.
```

Comment:
Please add examples for this in the example phapp.yml files, see examples directory.

Reply:
Please recheck. I keep stored now the manifest of respective sub-app once parsed.